.github/workflows/cypress.yaml

```diff
@@ -44,7 +44,11 @@ jobs:
             commit_message: ${{ steps.commit.outputs.message }}
             commit_author: ${{ steps.commit.outputs.author }}
             commit_email: ${{ steps.commit.outputs.email }}
-            percy_enable: ${{ steps.percy.outputs.value || '0' }}
+            # Percy is disabled while we're figuring out https://github.com/vector-im/wat-internal/issues/36
+            # and https://github.com/vector-im/wat-internal/issues/56. We're hoping to turn it back on or switch
+            # to an alternative in the future.
+            percy_enable: 0
+            # percy_enable: ${{ steps.percy.outputs.value || '0' }}
```

Comment:
Think this would make more sense happening by commenting out the percy step instead, otherwise it'd look confusing in the workflow run which says Enabling percy yet it doesn't

Reply:
Ah, yeah I hadn't thought of that. Agreed, that would be quite confusing. Have changed it as you proposed.